test/functional/bitcoin_cli.py

```diff
@@ -21,5 +21,15 @@ def run_test(self):
 
         assert_equal(cli_get_info, rpc_get_info)
 
+        user, password = get_auth_cookie(self.nodes[0].datadir)
+
+        self.log.info("Test -stdinrpcpass option")
+        assert_equal(0, self.nodes[0].cli('-rpcuser=%s' % user, '-stdinrpcpass', input=password).getblockcount())
+        assert_raises_process(1, "incorrect rpcuser or rpcpassword", self.nodes[0].cli('-rpcuser=%s' % user, '-stdinrpcpass', input="xpto").echo)
```

Comment:
Thanks for teaching me some Portuguese: https://pt.wikipedia.org/wiki/X.P.T.O. 🙂